cve_bin_tool/checkers/libxslt.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/python3
+
+"""
+CVE checker for libxslt
+https://www.cvedetails.com/product/14676/Xmlsoft-Libxslt.html?vendor_id=1962
+"""
+from . import Checker
+
+
+class LibxsltChecker(Checker):
+    CONTAINS_PATTERNS = []
+    FILENAME_PATTERNS = [r"libxlst.so."]
```

Comment:
`.` is a special character in the regex parser, so if you want those to be literally `.` instead of "match any 1 character" you'll need to use backslashes `r"libxslt\.so"`  (You're already doing it correctly in the VERSION_PATTERNS below.) 

Note also you've got a typo here of libxLst instead of xSlt, which might be why your tests are failing.